### PR TITLE
791 add fields to fieldofview in session

### PR DESCRIPTION
**Pull request by arielleleon**
Added new fields to the FieldOfView session object required for multiplane sessions.

### CODE REVIEW

File: src/aind_data_schema/core/session.py [hunk 1]
```diff
@@ -73,6 +77,14 @@ class FieldOfView(AindModel):
     frame_rate: Optional[Decimal] = Field(None, title="Frame rate (Hz)")
     frame_rate_unit: FrequencyUnit = Field(FrequencyUnit.HZ, title="Frame rate unit")
     coupled_fov_index: Optional[int] = Field(None, title="Coupled FOV", description="Coupled planes for multiscope")
+    power: Optional[Decimal] = Field(None, title="Power (mW)")
+    power_unit: Optional[PowerUnit] = Field(PowerUnit.MW, title="Power unit")
```
Comment (jtyoung84) [inline src/aind_data_schema/core/session.py:81]: I think we've been keeping the Unit fields as non-Optional and set with a default

File: src/aind_data_schema/core/session.py [hunk 2]
```diff
@@ -73,6 +77,14 @@ class FieldOfView(AindModel):
...
+    scanfield_z: Optional[int] = Field(
+        None,
+        title="Z stage position of the fastz actuator for a given targeted depth",
+    )
+    scanfield_z_unit: Optional[SizeUnit] = Field(None, title="Z stage position unit")
```
Comment (jtyoung84) [inline src/aind_data_schema/core/session.py:86]: I think we've been keeping the Unit fields as non-Optional and set with a default

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
The examples in the examples folder may need to be updated

File: src/aind_data_schema/core/session.py (hunk 2 above)
Comment (arielleleon) [reply src/aind_data_schema/core/session.py]: @jtyoung84  Got it and fixed

**Comment by arielleleon** [comment]
@jtyoung84 Examples have been added

**Review by jtyoung84 — APPROVED** [review]

**Comment by arielleleon** [comment]
After meeting with @saskiad , I fixed / added the following:

- using coupled_fov_index
- removed scanimage_resolution for the provided fov_scale_factor
- default power for fov set to PERCENT by default

File: examples/multiplane_ophys_session.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                    fov_coordinate_ml=1.5,
+                    fov_coordinate_ap=1.5,
+                    fov_coordinate_unit=SizeUnit.UM,
+                    fov_reference="Bregma",
+                    fov_width=512,
+                    fov_height=512,
+                    fov_size_unit=SizeUnit.UM,
+                    magnification="10x",
+                    fov_scale_factor=0.78,
+                    fov_scale_factor_unit=SizeUnit.UM,
```
Comment (saskiad) [inline examples/multiplane_ophys_session.py:79]: fov_scale_factor_unit should be um/pixel, correct?

**Review by saskiad — CHANGES_REQUESTED** [review]
Minor thing with the `fov_scale_factor_unit`

File: examples/multiplane_ophys_session.py (hunk 3 above)
Comment (arielleleon) [reply examples/multiplane_ophys_session.py]: Good catch - this is fixed now

**Review by saskiad — APPROVED** [review]

**Pull request merged**